File: Challenges/Coding World Cup/2023/Day 24 - NED vs BAN - Distinct String.java
import java.util.*;
import java.io.*;

public class Solution {
    static int simpleString(String s) {
        int n = s.length();
        int count = 0;

        for (int i = 1; i < n; i++) {
            if (s.charAt(i - 1) == s.charAt(i)) {
                count++;
                i++;
            }
        }

        return count;
    }
}

File: Challenges/Coding World Cup/2023/Day 24 - NED vs BAN - Distinct String.cpp
#include <bits/stdc++.h>
int simpleString(string s)
{
    // Write your code here
    int n = s.size();
    int count = 0;

    for (int i = 1; i < n; i++)
    {
        if (s[i - 1] == s[i])
        {
            count++;
            i++;
        }
    }

    return count;
}
